ports/nrf/boards/pca10100/mpconfigboard.mk

```diff
@@ -19,6 +19,10 @@ CIRCUITPY_DISPLAYIO = 0
 CIRCUITPY_FRAMEBUFFERIO = 0
 CIRCUITPY_FREQUENCYIO = 0
 CIRCUITPY_I2CPERIPHERAL = 0
+
+# Support for Internal Pullups for this MCU
+CIRCUITPY_I2C_INTERNAL_PULLUPS = 1
+
```

Comment:
The default for the `pca10100` should not be changed: you are using it as a test board, but that's for your own use. It is your company's board that will turn this on; no existing boards will do this.

Reply:
As above?